src/routes/admin.js

```diff
@@ -91,6 +92,17 @@ router.get('/bookmarks.db', isAuthenticated, async (req, res) => {
   res.download(filePath);
 });
 
+router.get('/bookmarks.csv', isAuthenticated, async (req, res) => {
+  const bookmarksDb = req.app.get('bookmarksDb');
+  const bookmarks = await bookmarksDb.getBookmarksForCSVExport();
+  const result = csvStringify(bookmarks);
```

Comment:
csv-stringify does also have an streaming API such that we could asynchronously stringify a row and then write that to the client. I didn't do that because this was simpler and we _probably_ don't need to worry about memory ballooning during an export right? That would be _so many_ bookmarks :grimacing: